sql/_31_cherry/issue_21506_online_index/answers/cbrd_22400.answer

```diff
@@ -9,17 +9,21 @@
 ===================================================
 Error:-670
 ===================================================
+0
+===================================================
 Table    Non_unique    Key_name    Seq_in_index    Column_name    Collation    Cardinality    Sub_part    Packed    Null    Index_type    Func    Comment    Visible    
-dba.t1     0     pk_t1_a     1     a     A     0     null     null     NO     BTREE     null     null     YES     
+dba.t1     0     pk_t1_a     1     a     A     2     null     null     NO     BTREE     null     null     YES     
 
 ===================================================
 1
 ===================================================
 0
 ===================================================
+0
+===================================================
 Table    Non_unique    Key_name    Seq_in_index    Column_name    Collation    Cardinality    Sub_part    Packed    Null    Index_type    Func    Comment    Visible    
 dba.t1     0     idx_t1_b     1     b     D     1     null     null     YES     BTREE     null     null     YES     
-dba.t1     0     pk_t1_a     1     a     A     1     null     null     NO     BTREE     null     null     YES     
+dba.t1     0     pk_t1_a     1     a     A     2     null     null     NO     BTREE     null     null     YES     
```

Comment:
after deleting 'a=1' cardinality didn't change.

Reply:
@shparkcubrid 
Is this result correct after delete 'a=1' ?

Reply:
If page didnt change, the update statistics doesnt work well caused by logic.
so , need to use 'update statics... with fullscan'.